test/torch/modules/test_torch_distribution_inference.py

```diff
@@ -268,8 +268,13 @@ def test_poisson(rate: float) -> None:
     ), f"rate did not match: rate = {rate}, rate_hat = {rate_hat}"
 
 
-# The following parameters match the Gluon-based test, adjusted for the different parametrization
-@pytest.mark.parametrize("total_count, logit", [(1 / 0.7, np.log(2.5 * 0.7))])
```

Comment:
Any particular reason these have been changed?

Reply:
Mostly readability. There was a bug in the `NegativeBinomialOutput` which went unnoticed because the logit was passed were a probability was required: it just happens that `np.log(2.5 * 0.7)) ≈ 0.56` so it's both valid as a logit and a probability. So I just replaced this test case parameters with explicit numbers, and added another one where the logit is clearly *not* a probability